### PR TITLE
feat: add RUN_SUPERSET config

### PR DESCRIPTION
**Pull request by Ian2012**
## Description

This PR:

- Adds support for `RUN_SUPERSET` config
- Use the `SUPERSET_PORT`  to use with the caddy server

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by itsjeyd** [comment]
@Ian2012 Thanks for this contribution! Let us know when it is ready for review.

**Review by bmtcril — APPROVED** [review]
Seems good to me, out of curiosity why was `tutorsuperset/patches/local-docker-compose-dev-jobs-services` removed?

File: tutorsuperset/patches/local-docker-compose-dev-jobs-services [hunk 1]
```diff
@@ -1,7 +0,0 @@
-superset-job:
-  {% include 'base-docker-compose-services' %}
-    OPENEDX_LMS_ROOT_URL: "http://{{ LMS_HOST }}:8000"
-  depends_on:
-    - mysql
-    - redis
-    - superset
```
[the remark this answers is not in the document]
Comment (Ian2012) [reply tutorsuperset/patches/local-docker-compose-dev-jobs-services:7]: This patch do not exists in tutor, the correct one is this: https://docs.tutor.overhang.io/reference/patches.html#local-docker-compose-jobs-services and we already use it

**Comment by Ian2012** [comment]
@mariajgrimaldi @bmtcril  This patch do not exists in tutor, the correct one is this: https://docs.tutor.overhang.io/reference/patches.html#local-docker-compose-jobs-services and we already use it

**Comment by Ian2012** [comment]
@itsjeyd It's ready for review

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-superset&entry.867055334=https://github.com/openedx/tutor-contrib-superset/pull/32&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-05-08+19:08&entry.1917517419=2023-05-09+17:14&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**